github/scan.yaml

```diff
@@ -0,0 +1,57 @@
+env:
+    SYSDIG_SECURE_ENDPOINT: "https://secure.sysdig.com"
+    REGISTRY_HOST: "quay.io"
+    IMAGE_NAME: "mytestimage"
+    IMAGE_TAG: "my-tag"
+  
+name: Container build, scan and push
+
+on: [push, pull_request]
+
+jobs:
+  scan:
+    runs-on: ubuntu-latest
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v2
+    
+    - name: Set up Docker Buildx
+      uses: docker/setup-buildx-action@v2
+    
+    - name: Build and save
+      uses: docker/build-push-action@v3
+      with:
+        context: github/
+        tags: ${{ env.REGISTRY_HOST }}/${{ secrets.REGISTRY_USER }}/${{ env.IMAGE_NAME }}:${{ env.IMAGE_TAG }}
+        load: true
+
+    - name: Download sysdig-cli-scanner if needed
+      run:  |
+        curl -sLO https://download.sysdig.com/scanning/sysdig-cli-scanner/latest_version.txt
+        mkdir -p /tmp/{db,cache}
```

Comment:
Do we need to keep the latest_version.txt in /tmp/cache? I kind of understand using /tmp/db for the vulnerability mini DB and using the github action cache feature to persist between executions, but what about the latest_version.txt? I think the installation would like simpler using the one-liner provided in the docs.

Reply:
My plan is to create a follow up PR to leverage cache for both the databases and the binary itself, so if the binary has been already downloaded and is the same version it doesn't need to be downloaded again.